_inc/client/state/rewind/reducer.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import get from 'lodash/get';
+import assign from 'lodash/assign';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_STATUS_FETCH,
+	REWIND_STATUS_FETCH_RECEIVE,
+	REWIND_STATUS_FETCH_FAIL,
+} from 'state/action-types';
+
+export const data = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case REWIND_STATUS_FETCH_RECEIVE:
+			return assign( {}, state, { action.siteData } );
```

Comment:
This won't work. The thing you tried to do here, just works with top level vars (If it was just `siteData` it would work, but not when the var is an object property). Should read `{ siteData: action.siteData }`
Another solution would be to destructure the `action` arg like:

```
export const data = ( state = {}, { type, siteData } ) => {
 	switch (type ) {
 		case REWIND_STATUS_FETCH_RECEIVE:
 			return assign( {}, state, { siteData } );
```